src/zenml/hub/mingpt_example/mingpt/utils.py

```diff
@@ -0,0 +1,116 @@
+import json
+import os
+import random
+import sys
+from ast import literal_eval
+
+import numpy as np
+import torch
+
+# -----------------------------------------------------------------------------
```

Comment:
Just a small thing that probably needs to be deleted

Reply:
Yes all these things relating to the LLM quickstart will have to go somewhere. I'm not sure where. Maybe a separate repo for now, or into a branch on projects so it can be used for the LLM project?